Tasks/PublishTestResultsV2/make.json

```diff
@@ -2,7 +2,7 @@
   "externals": {
     "archivePackages": [
       {
-        "url": "https://testmanagementstore.blob.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
+        "url": "https://testmanagementstore.z13.web.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
```

Comment:
We might have to update the signing pipeline first, to ensure that we are able to publish a latest version of PTR library in this new location first.

Reply:
Updated Release as well

Reply:
https://dev.azure.com/mseng/AzureDevOps/_releaseProgress?releaseId=21585947&_a=release-task-editor&environmentId=212631715